pkg/config/containers.conf

```diff
@@ -386,6 +378,22 @@
 # Number of seconds to wait for container to exit before sending kill signal.
 # stop_timeout = 10
 
+# Index to the active service
+# active_service = production
+
+# map of service destinations
+# [service_destinations]
+#   [service_destinations.production]
+#     URI to access the Podman service
+#     Examples:
+#       rootless "unix://run/user/$UID/podman/podman.sock" (Default)
+#       rootfull "unix://run/podman/podman.sock.(Default)
+#       remote rootless ssh://engineering.lab.company.com/run/user/1000/podman/podman.sock
+#       remote rootfull ssh://root@10.10.1.136:22/run/podman/podman.sock
+#     uri = ""
+#     Path to file containing ssh identity key
+#     identity = ""
```

Comment:
I would like to had an actual example
#  [service_destinations.production]
#          uri="ssh://engineering.lab.company.com/run/user/1000/podman/podman.sock"
#          idenitry="~/.ssh/id_rsa"

Reply:
Could you add a function that does the following.
```
func (*c config) Destination(system string) { uri string, identity string) {
                     If system == "" {
                         If c.Engine.Active == "" {
                            return 	c.Engine.RemoteURI, c.Engine.RemoteIdentify
                         } 
                         server = c.Engine.Active
                   }
                   return c.Engine.ServiceDestinations[server].URI, c.Engine.ServiceDestinations[server].identity 
   } 
```